### DISCUSSION TITLE
How can I prevent it from outputting a blank line at startup while allowing a blank line between commands?

### DISCUSSION THREAD

**Discussion by sketcch** [starship/starship#5135] [Q&A] [answered]
I want to prevent blank lines from being output during startup, but have blank lines between commands. I referred to the official configuration documentation and found that the configuration for both is the same. This means either there will be blank lines at startup and between commands, or there won't be any blank lines at all. How can I achieve no blank lines during startup, but have blank lines between commands?

Configuration documentation:
![image](https://user-images.githubusercontent.com/116141481/234168590-27327209-7b32-4ed5-b778-3111320d5996.png)
![image](https://user-images.githubusercontent.com/116141481/234168608-de1fa9b0-df95-4ba1-83ad-95bae8301717.png)

When add_newline = true
![image](https://user-images.githubusercontent.com/116141481/234168884-a52d21d7-2896-40a5-8cc0-becf40672c12.png)

When add_newline = false
![image](https://user-images.githubusercontent.com/116141481/234168933-420c5de2-6d05-47f1-8048-3d0438aea9ac.png)

**Comment by mickimnet** [comment, marked as answer]
Currently not possible. There is an open pull request, for it–see: #5072
